dask/bytes/core.py

```diff
@@ -19,8 +19,9 @@
 _open_text_files = dict()
 
 
-def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
-                     sample=True, compression=None, **kwargs):
+def read_bytes(urlpath, delimiter=None, not_zero=False, blocksize=2**27,
+                     sample=True, compression=None, storage_options=None,
+                     **kwargs):
```

Comment:
@frol any thoughts on just using kwargs here?  I don't see the reason to keep around a separate dict of keyword arguments at this stage.  It is unambiguous where they will end up so it seem simpler to just keep one set around.